### PR TITLE
Themed Icons

### PR DESCRIPTION
**Pull request by sithlord48**
- Fix Crash if walkmesh renders without a valid gpuRenderer
 - Use Psx Icons from ff7tk (based on theme)
 - Use Theme icons for actions where possible
 - Embed Basic Light / Dark icons from [breeze](https://github.com/KDE/breeze-icons)

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -509,22 +505,20 @@ endif()
 
 # GUI
 if(GUI)
-    qt_add_executable(${GUI_TARGET} MANUAL_FINALIZATION MACOSX_BUNDLE WIN32 ${PROJECT_SOURCES} ${QM_FILES} ${RESOURCES} ${EXTRA_RESOURCES_GUI})
+    qt_add_executable(${GUI_TARGET} MANUAL_FINALIZATION MACOSX_BUNDLE WIN32 ${PROJECT_SOURCES} ${QM_FILES} ${RESOURCES} ${EXTRA_RESOURCES_GUI} icons/icons.qrc)
```
Comment (myst6re) [inline CMakeLists.txt:508]: Please add icons/icons.qrc in the RESOURCES variable

Comment (sithlord48) [reply CMakeLists.txt:508]: We don't need or want them for the cli target.

File: src/3d/WalkmeshWidget.cpp [hunk 2]
```diff
@@ -42,7 +42,8 @@ void WalkmeshWidget::clear()
 	fieldModels.clear();
 	update();
 
-	gpuRenderer->reset();
+	if(gpuRenderer)
```
Comment (myst6re) [inline src/3d/WalkmeshWidget.cpp:45]:
```suggestion
	if (gpuRenderer) {
```

**Comment by myst6re** [comment]
Thank you very much!

**Pull request merged**